driver/wnbd_dispatch.c

```diff
@@ -190,6 +189,35 @@ NTSTATUS WnbdDispatchRequest(
 
             RtlCopyMemory(Buffer, SrbBuffer, Element->DataLength);
             break;
+        case WnbdReqTypeFlush:
+            Request->Cmd.Flush.BlockAddress =
+                Element->StartingLbn / DevProps->BlockSize;;
```

Comment:
nit: s/;;/;/

Reply:
Whops, thanks.

Reply:
nit: s/;;/;/g

The first occurrence from `WnbdReqTypeRead` got fixed but there is still one in `WnbdReqTypeFlush`.